gui/public/i18n/en/translation.ftl

```diff
@@ -424,6 +424,9 @@ settings-general-interface-dev_mode = Developer Mode
 settings-general-interface-dev_mode-description = This mode can be useful if you need in-depth data or to interact with connected trackers on a more advanced level.
 settings-general-interface-dev_mode-label = Developer Mode
 settings-general-interface-theme = Color theme
+settings-general-interface-show-navbar-onboarding = Show setup wizard on navigation bar
+settings-general-interface-show-navbar-onboarding-description = This changes if the "Setup Wizard" button shows on the navigation bar.
```

Comment:
instead of "Setup Wizard" maybe you want to reference the translation key itself?
you can do that by doing `{ navbar-onboarding }`